controllers/auth.js

```diff
@@ -0,0 +1,60 @@
+import db from '../models';
+import Auth from '../helpers/auth';
+
+export const signup = async (req, res) => {
+  const {
+    username, firstName, lastName, email,
+  } = req.body;
+  const password = Auth.hashPassword(req.body.password);
```

Comment:
1. destructure password in the request body on line `6`
2. rename `password` to `hashedPassword`